src/announcements/types.ts

```diff
@@ -3,6 +3,7 @@ import {FirebaseFirestoreTypes} from '@react-native-firebase/firestore';
 
 export type AnnouncementRaw = {
   id: string;
+  active: boolean;
```

Comment:
Do we need the field here since we use `active == true` in the query?

Reply:
Belte og bukseseler, er det ikke det de kaller det?